lxd/acme/acme.go

```diff
@@ -87,6 +88,12 @@ func UpdateACMECertificate(s *state.State, provider HTTP01Provider, clustered bo
 			break
 		}
 
+		// In case we were rate limited, don't try again.
```

Comment:
The commit " lxd/acme: Don't retry if rate limited " should be rebased into the commit that adds the new function `UpdateACMECertificate`